enhancements/two-node-fencing/tnf.md

```diff
@@ -687,76 +781,94 @@ Additionally, it would be good to have workload-specific testing once those are
+
+This option requires the user to run external software like keepalived in front of the instances to ensure that only one instance is active and receiving traffic at a time.
+
+The advantage of this this kind of set up is that failures of each instance is reliably deterministic. A failure of your active instance just passes traffic over to your passive instance, and a
+failure of your passive instance doesn't interrupt the active instance at all. Since both instances cannot be active at the same time, there is no risk of a split-brain situation. Additionally, each
+Single Node OpenShift instance is a cluster by itself - meaning it does need the other instance to be operational for it to proceed with upgrade or configuration operations that require a node reboot.
+
+That said, there are major tradeoffs of structuring the nodes in the fashion. Firstly, all aspects of the customer's applications must be available on both nodes. This means that the cluster
+administrator needs to actively synchronize both nodes so that either of them could become the active node at any given time. How this data is synchronized between the SNO instances is left as an a
+challenge for the solutions architect. For some applications, this results in more overhead on each node since both need to be able to run fully independently. Workloads that expect to run multiple
+instances in a single cluster will need to run all of their workload instances on both clusters, and may need special configuration to synchronize data between the active and passive clusters.
+Additionally, the end user cannot utilize the full capacity of both nodes at the same time.
+
+The bottom line is that this solution may be viable for simple setups with stateless workloads, but it becomes logistically much harder to deploy and maintain for workloads that need to maintain
+synchronised state between the OpenShift instances. Solving the state synchronization challenges means having to solve the same core issues TNF faces around maintaining data integrity - without
+actively benefiting from the computation power of the second server.
+
+#### MicroShift
+MicroShift was considered as an alternative but it was ruled out because it does not support multi-node and has a very different experience than OpenShift which does not match the TNF initiative which
+is on getting the OpenShift experience on two nodes
+
+#### 2 SNO + KCP:
+[KCP](https://github.com/kcp-dev/kcp/) allows you to manage multiple clusters from a single control-plane, reducing the complexity of managing each cluster independently. With kcp, you can manage the
+two single-node clusters, each single-node OpenShift cluster can continue to operate independently even if the central kcp management plane becomes unavailable. The main advantage of this approach is
+that it doesn’t require inventing a new Openshift flavor and we don’t need to create a new installation flow to accommodate it.
```

Comment:
How would workloads be affected?
For example, if I'm a deployment with replica=2, would it be possible to arrange for 1 replica to be on each SNO?

Reply:
That's not really possible. I tried to address this point in a sentence about workloads with multiple instances, but I'll be even more specific.